.storybook/webpack.config.ts

```diff
@@ -0,0 +1,16 @@
+import { Configuration } from 'webpack';
+import { clientConfig } from '../webpack.config';
+
+export default async ({ config } : { config: Configuration }) : Promise<Configuration> => {
+	return {
+		...config,
+		target: clientConfig?.target,
+		resolve: clientConfig?.resolve,
+		module: {
+			rules: clientConfig?.module?.rules ?? [],
+		},
+		plugins: [
+			...config.plugins ?? []
```

Comment:
If `config.plugins` is undefined will this result in `[[]]`?

Reply:
I was puzzled by this a bit too but what it actually does is something like `...(config.plugin ?? [])`